workspaces/web/src/components/ChallengePanel.svelte

```diff
@@ -52,6 +52,7 @@
 
   .panel {
     position: fixed;
+    min-height: 12vh;
```

Comment:
Why is min-height necessary?

Reply:
The updated design present here https://github.com/kantord/LibreLingo/issues/81#issuecomment-601997442, increase the height. Currently height wraps around the component, to increase some more negative space, there is a need for min-height.